### ISSUE TITLE
How to send data to the model

### ISSUE THREAD

**Issue by DrBubbles42** [charlieforward9/visualPT#43]

**Comment by charlieforward9** [comment]
I will create some simple documentation on how we capture camera data.

**Comment by DrBubbles42** [comment]
I think the best way to do this would be to reduce the size of the image. The time it takes to train a model will be dependent on the number of parameters it has or in our case the resolution of the image. The lower the resolution the faster everything will be but we dont want to go too low because we will lose out on features within the data.

**Comment by DrBubbles42** [comment]
An Idea that I had would be to draw a box on the app to put the subject in when filming. This would provide some continuity for the model. Each subject would be the same distance from the camera and in the center of the frame. This will increase our accuracy and allow us to trim the images down a bit before passing them to the model.

**Comment by charlieforward9** [comment]
> I think the best way to do this would be to reduce the size of the image.

I have some control over the resolution of the image from the front end, and remember that I already have the pose landmark output, the 33 points described in the Mediapipe pose detection model card. We could pass these landmarks in instead of images, which would significantly reduce the size of the inputs. 

TLDR; What do you think about using landmarks to train the balance model instead of the raw images? 

![image](https://github.com/VisualPT/visualPT/assets/62311337/9c973891-b9bf-44a7-aee2-c44774eecac6)

**Comment by DrBubbles42** [comment]
you already have this implemented!? This will be a better than the images! I thought we would have to do something like this!

**Comment by charlieforward9** [comment]
Sample Detection Snippet that already exists in the application.

![image](https://github.com/VisualPT/visualPT/assets/62311337/1cdbd69d-4201-4f54-8781-492e99074038)

**Comment by DrBubbles42** [comment]
Yes we will use this instead of the images

**Comment by charlieforward9** [comment]
Fantastic, see #45. I am brainstorming how to label the data from the UI. This is what I have so far for you. Should help prevent tons of manual labeling.

**Comment by DrBubbles42** [comment]
The model will take the data from the landmark wireframe output. The input to our model will be the output from the OpenCV pose detection model which is 33x5. this is much smaller than 255x255x3 so this is perfect and definitely the way to go.

The model will take a 33x5 array as input and output a value between 0 and 1 where 0 is fail and 1 is success.
